build-tests/heft-sass-test/src/styles.sass

```diff
@@ -4,10 +4,10 @@
  */
 
 // Testing Sass imports
-@import 'stylesImport'
+@use 'stylesImport'

```

Comment:
I'm assuming based on lower lines of code that both `@use` and `@import` are supported? I think it would be prudent for us to have a test supporting both formats so that we cover both scenarios (especially since the change file depicts this change as SemVer minor)

Reply:
Thank you to review and I agree with this. Would you please review the last 2 commits?